tools/packaging/debian/dragonfly.conf

```diff
@@ -1,4 +1,5 @@
 --pidfile=/var/run/dragonfly/dragonfly.pid
 --log_dir=/var/log/dragonfly
 --dir=/var/lib/dragonfly
+--max_log_size=20
```

Comment:
I dont find this flag.. where is it?

Reply:
`./dragonfly --helpfull |  less` it's part of glog